src/components/molecules/OakAccordion/OakAccordion.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ReactNode, useState } from "react";
+import styled from "styled-components";
+
+import { OakBox, OakFlex, OakIcon } from "@/components/atoms";
+import { parseSpacing } from "@/styles/helpers/parseSpacing";
+import { parseDropShadow } from "@/styles/helpers/parseDropShadow";
+
+type OakAccordionProps = {
+  /**
+   * The header of the accordion
+   */
+  header: ReactNode;
+  /**
+   * Slot to place content after the header and outside the button
+   */
+  headerAfterSlot?: ReactNode;
+  /**
+   * Whether the accordion should be open initially
+   */
+  initialOpen?: boolean;
+  /**
+   * The content of the accordion
+   */
+  children: ReactNode;
+  /**
+   * The id of the accordion
+   */
+  id: string;
+};
+
+const StyledOakFlex = styled(OakFlex)`
+  font: inherit;
+  border: none;
+  background: none;
+  appearance: none;
+  margin: -${parseSpacing("inner-padding-m")};
+
+  outline: none;
+
+  &:focus-visible {
+    box-shadow: ${parseDropShadow("drop-shadow-centered-lemon")},
+      ${parseDropShadow("drop-shadow-centered-grey")};
+  }
+`;
+
+/**
+ * An accordion component that can be used to show/hide content
+ */
+export const OakAccordion = ({
+  header,
+  headerAfterSlot,
+  children,
+  initialOpen = false,
+  id,
+}: OakAccordionProps) => {
+  const [isOpen, setOpen] = useState(initialOpen);
```

Comment:
Accordions do not currently support automatic closing of their siblings. This is something that could be added later with an `OakAccordionGroup` component and a shared context. Right now I don't need this for the cookie banner work but it's a small enhancement for someone to add later if/when it is needed.